blocks/library/embed/index.js

```diff
@@ -39,6 +39,18 @@ function getEmbedBlockSettings( { title, icon, category = 'embed' } ) {
 			caption: children( 'figcaption' ),
 		},
 
+		transforms: {
+			from: matchers.map( ( regExp ) => ( {
```

Comment:
While current direction of #1902 is to rename all references of "query" and "matchers" to "source", do you think we still ought to name this something more... unique? Like `patterns` ?

[(parameter to `RegExp` constructor)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp)